File: practice/palindrome.py
# NUMBER IS PALINDROME OR NODE

n = 12122
original_n = n
rev = 0
while n:
    rev = rev*10 + n%10
    n //= 10
print("Reverse is: ", rev)

if original_n == rev:
    print("number is palindrome")
else:
    print("number is not palindrome")

File: practice/palindrome.c
// NUMBER IS PALINDROME OR NODE

#include<stdio.h>
#include<stdlib.h>
int main ()
{
    int n = 2121, rev = 0;
    int  original_n = n;
    while (n != 0)
    {
        rev = rev*10 + n%10;
        n /= 10;
    }
    printf("reversed number = %d\n", rev);

    if(rev == original_n){
        printf("Palindrome");
    }else{
        printf("Not palindrome");
    }
    
    return 0;
}